start.html

```diff
@@ -99,7 +99,49 @@
 how to find elements on the page. WebDriver offers a number of built-in selector
 types, amongst them finding an element by its ID attribute:
 
-<pre><code class=java>WebElement cheese = driver.findElement(By.id("cheese"));</code></pre>
+    <div class="tabs">
+        <ul class="tabs">
```

Comment:
Drop outer `<div>` and rename from “tabs” to maybe “languages”?

On a more general note I think this list can automatically be generated and inserted with JavaScript in the future.